pkg/client/k8s.go

```diff
@@ -482,9 +485,60 @@ func (k K8sClient) DeleteBatchJob(name string, namespace string) error {
 	return k.K8sCS.BatchV1().Jobs(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
 }
 
-/*
-	LOCAL VOLUMES SPECIFIC METHODS
-*/
+// GetPodLogs returns logs from the containers running in the pod
+func (k K8sClient) GetPodLogs(podName string, ns string) string {
+	req := k.K8sCS.CoreV1().Pods(ns).GetLogs(podName, &corev1.PodLogOptions{})
+	podLogs, err := req.Stream(context.TODO())
+	if err != nil {
+		log.Fatal("err getting logs ", err)
+	}
+	defer podLogs.Close()
+
+	buf := new(bytes.Buffer)
+	_, err = io.Copy(buf, podLogs)
+	if err != nil {
+		log.Fatal("err converting log buffer", err)
+	}
+
+	return buf.String()
+}
+
+// StartPodLogsStream starts stream of logs for the given pod
+func (k K8sClient) StartPodLogsStream(podName string, ns string) {
```

Comment:
Left this function to be used for streaming logs in some upcoming PR. It is not used rn.